bulbs/api/management/commands/create_app_authtoken.py

```diff
@@ -0,0 +1,38 @@
+import random
+
+from django.contrib.auth import get_user_model
+from django.core.management.base import BaseCommand
+
+from rest_framework.authtoken.models import Token
+
+
+User = get_user_model()
+
+
+class Command(BaseCommand):
+
+    help = "Creates a new user & app token for a client request."
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '-username',
+            dest='username',
+            help='The username for the given application.',
+            required=True,
+            type=str
+        )
+
+    def handle(self, *args, **kwargs):
+        username = kwargs.get('username')
+        qs = User.objects.filter(username=username)
+        if not qs.exists():
+            user = User.objects.create(
+                username=username,
+                is_staff=True,
+                password=random.getrandbits(128)
```

Comment:
random library isn't cryptographically secure, better to use something like os.urandom or SystemRandom

http://stackoverflow.com/questions/20936993/how-can-i-create-a-random-number-that-is-cryptographically-secure-in-python